e2e-tests/constants/routes/policy-and-export.js

```diff
@@ -3,6 +3,8 @@ const ROOT = '/policy-and-exports';
 export const POLICY_AND_EXPORTS = {
   ROOT,
   TYPE_OF_POLICY: `${ROOT}/type-of-policy`,
+  SINGLE_CONTRACT_POLICY: `${ROOT}/single-contract-policy`,
```

Comment:
I just wonder in the future, is it worth doing quote root and insurance root so we know which is which on quick glance?

Reply:
Good shout - this file should also be in `routes/insurance/type-of-policy` I think, will have a look